README.md

```diff
@@ -19,5 +20,35 @@ Note that because of a documented bug in libunwind `unw_getcontext`
 leads to a page fault, and in turn `backtrace_full`will also lead to
 one as it uses `_Unwind_Backtrace` which calls `unw_getcontext`.
 
+For libgcov you will need the following libraries that should be added to the
+`Makefile` in this order:
+
+* `pthreads`, e.g `pthread-embedded`
+* `compiler-rt`
+* `libcxx`
+* `libcxxabi`
+* `libunwind`
+* `libc`, e.g. `newlib`
+
+In order to use the `PGO` mechanism, the `fprofile-generate` and `fprofile-use`
+flags should be added to the `Makefile.uk` of the application running in Unikraft.
+
+For example, if you want to use the `PGO` mechanism for the `helloworld`
+application you have to add the following lines in the `Makefile.uk`:
+
+`APPHELLOWORLD_CFLAGS -$(CONFIG_OPTIMIZE_PGO_GENERATE)          += -fprofile-generate`
+
+`APPHELLOWORLD_CFLAGS -$(CONFIG_OPTIMIZE_PGO_USE)               += -fprofile-use`
```

Comment:
Add these a single block of code. Use triple backticks used in Markdown[1].

[1] https://guides.github.com/features/mastering-markdown/